examples/step-19/step-19.cc

```diff
@@ -697,19 +660,9 @@ namespace Step19
             dim>::particle_iterator_range particles_in_cell =
             particle_handler.particles_in_cell(cell);
 
-          const unsigned int n_particles_in_cell =
-            particle_handler.n_particles_in_cell(cell);
-
-          particle_positions.resize(n_particles_in_cell);
-          {
-            typename Particles::ParticleHandler<dim>::particle_iterator
-              particle = particles_in_cell.begin();
-            for (unsigned int particle_index = 0;
-                 particle != particles_in_cell.end();
-                 ++particle, ++particle_index)
-              particle_positions[particle_index] =
-                particle->get_reference_location();
-          }
+          particle_positions.clear();
```

Comment:
Yes, I will.

Reply:
Thinking about it, I think I will also skip the `reserve` because that seems like a micro-optimization at this point, now that I'm initializing an empty variable and using `push_back` into it.